### ISSUE TITLE
[FEATURE 💡] Tags must be added

### ISSUE THREAD

**Issue by warmachine028** [warmachine028/memories#82]
### Description

- have a look at this [previous deployment](https://memories-5xye0f5aj-warmachine028.vercel.app).
- Previously tags could be added in the Form by Chipinput.js
- Tags were searchable too.
- After migrating to MUI v5, we had to drop support for [mui-chip-input](https://www.npmjs.com/package/mui-chips-input).
- We used our own Component at Chipinput.js.
- But after migrating to Vite from CRA at #72 , we had to disable support for ChipInput.jsx
- Bring back this feature in the App, after using this feature carefully in the previous deployment.

### Screenshots

- Previously in Memories
![image](https://github.com/warmachine028/memories/assets/75939390/cb63b2c8-3249-4790-bf7f-35971f9d5f81)

- MUI Chip Input
![image](https://github.com/warmachine028/memories/assets/75939390/813b698c-9431-453a-a5fc-a15ebf1491aa)

### Additional information

- Always use styled components instead of plain CSS and MUI components.
## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

**Comment by TOLU-MICH** [comment]
hi, please can you assign me to the project

**Comment by warmachine028** [comment]
alright, assigning you

**Comment by warmachine028** [comment]
@TOLU-MICH what's the progress on this?

**Comment by Faisalhs51** [comment]
@warmachine028 Please assign me this

**Comment by warmachine028** [comment]
Assigning you

**Comment by warmachine028** [comment]
Yes can you implement the chip input from mui's existing components?

**Comment by Faisalhs51** [comment]
I'll give it a try

**Comment by warmachine028** [comment]
Pull the Latest changes in branch to avoid merge conflicts
